actions/nuxeo-add-to-collection-button.html

```diff
@@ -53,8 +50,11 @@
 
     <dom-if if="[[_isAvailable(document)]]">
       <template>
-        <paper-icon-button icon="[[icon]]" on-click="_toggleDialog" noink></paper-icon-button>
-        <nuxeo-tooltip>[[i18n('addToCollectionButton.tooltip')]]</nuxeo-tooltip>
+        <div class="action" on-click="_toggleDialog">
+          <paper-icon-button icon="[[icon]]" noink></paper-icon-button>
+          <span class="label" hidden$="[[!showLabel]]">[[_label]]</span>
+          <nuxeo-tooltip>[[_label]]</nuxeo-tooltip>
```

Comment:
should tooltip also handle click and toggle dialog?

Reply:
Don't think we want that. Will fix it. 👍